Libraries/NativeAnimation/Nodes/RCTInterpolationAnimatedNode.m

```diff
@@ -9,27 +9,72 @@
 
 #import "RCTAnimationUtils.h"
 
+NSRegularExpression *regex;
```

Comment:
If it actually has to be a global var (I think it could local as well -- it's just a regex), can't we initialize it immediately? 
Also, the name is not very clear.

Reply:
True, was mostly thinking it can be reused across instances, so made it shared among them.

Reply:
@osdnk I think it's not possible to initialize global vars like this in obj-c, or do you know some way?

Reply:
Improved it to use dispatch_once